src/Markdig/Extensions/Alerts/AlertBlock.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Alexandre Mutel. All rights reserved.
+// This file is licensed under the BSD-Clause 2 license.
+// See the license.txt file in the project root for more information.
+
+using Markdig.Helpers;
+using Markdig.Syntax;
+
+namespace Markdig.Extensions.Alerts;
+
+/// <summary>
+/// A block representing an alert quote block.
+/// </summary>
+public class AlertBlock : QuoteBlock
+{
+    /// <summary>
+    /// Creates a new instance of this block.
+    /// </summary>
+    /// <param name="kind"></param>
+    public AlertBlock(StringSlice kind) : base(null!)
```

Comment:
Instead of the `null!` should https://github.com/xoofx/markdig/blob/fcb56fb03701cc1a07a52769908579adb8927ee2/src/Markdig/Syntax/QuoteBlock.cs#L22 be updated to allow `null`?

Especially as https://github.com/xoofx/markdig/blob/fcb56fb03701cc1a07a52769908579adb8927ee2/src/Markdig/Syntax/Block.cs#L23 allows `null`.

Reply:
Fixed by commit 64dc4e6